src/util/constants.js

```diff
@@ -19,17 +9,25 @@ export const defaultExerciseSites = [
     "subdomain": "web",
     "name": "Hellotalk"
   },
+  {
+    "hostname": "www.zeeguu.org",
+    "href": "https://www.zeeguu.org/practice/#!/practice/plan/0",
+    "pathname": "//practice/#!/practice/plan/0",
+    "regex": "*://*.www.zeeguu.org/*",
+    "tld": "org",
+    "domain": "zeeguu",
+    "subdomain": "www",
+    "name": "Zeeguu"
+  }
   // 'https://www.brainscape.com/'
 ];
 
 export const defaultExerciseSite = defaultExerciseSites[0];
 
-export const defaultexerciseDuration = 5; // minutes
-
-export const exerciseDuration = 5 * 60 * 1000; // 5 minutes, in ms
+export const defaultexerciseDuration = 5 * 60 * 1000; // 5 minutes
 
 export const defaultTimeout = 5 * 60 * 1000; // 5 minutes
 
-export const defaultTimeoutInterval = 5 * 60 * 1000; // increase by a minute
+export const defaultTimeoutInterval = 5 * 60 * 1000; // 5 minute- increments
```

Comment:
timeout could stay 5 minutes for now, at least until we add the possibility of editing this value.
even for me, 30seconds / 30seconds is a bit frustrating!

Reply:
Not sure what you mean - do not this refers to this section:


![Screenshot_2019-07-09_15-43-15](https://user-images.githubusercontent.com/744430/60892999-5fd70d00-a260-11e9-9413-a0faef409885.png)

![Screenshot_2019-07-09_15-43-15](https://user-images.githubusercontent.com/744430/60893039-73827380-a260-11e9-8481-53e56b7a364e.png)

What do you mean with 30 sec / 30 sec? If this is a feature request please post a new issue, since this is a bugfix PR.